.github/workflows/release.yaml

```diff
@@ -98,6 +98,17 @@ jobs:
           build-args: VER=${{ needs.pre_release.outputs.release_tag }}
           platforms: linux/amd64,linux/arm64
 
+      - name: Build and push with base path
```

Comment:
Is it absolutely necessary to have a separate image for that?
From the first look it seems that just setting env variable should trigger the behavior. We have the ways to communicate configured base path from backend to frontend (via `StatusInfo` struct).

Reply:
I'm +1 on having only one image and have  configurable base path. I could not do it that way becasue `VITE_BASE_PATH` is only accesible at build time AFAIK. Once the image is built, we can only customize the base path for the backend via env var or as a  flag as you suggest.

Reply:
After reading this doc part: https://vite.dev/guide/build.html#relative-base
I have a feeling that we could just use 
```js
export default defineConfig({
  base: './' 
})
```
And it would do the trick universally. Did you try investigating that direction?